backend/api/Areas/Project/Controllers/ReportController.cs

```diff
@@ -126,7 +128,16 @@ public IActionResult GetProjectReportSnapshots(int reportId)
         [SwaggerOperation(Tags = new[] { "project" })]
         public IActionResult AddProjectReport(ProjectReportModel model)
         {
-            var report = _pimsService.ProjectReport.Add(_mapper.Map<Entity.ProjectReport>(model));
+            Entity.ProjectReport report;
+
+            if (model.Snapshots?.Any() == true)
```

Comment:
If snapshots are provided, create a report with them.

Reply:
is there a need for == true?